src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AzureBillingProjectWizard.test.ts

```diff
@@ -0,0 +1,157 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import { h } from 'react-hyperscript-helpers'
+import Events from 'src/libs/events'
+import {
+  addUserAndOwner, getAddUsersRadio,
+  getNoUsersRadio
+} from 'src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AddUsersStep.test'
+import {
+  AzureBillingProjectWizard, userInfoListToProjectAccessObjects
+} from 'src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AzureBillingProjectWizard'
+import {
+  selectManagedApp
+} from 'src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AzureSubscriptionStep.test'
+import {
+  clickCreateBillingProject,
+  nameBillingProject, verifyCreateBillingProjectDisabled
+} from 'src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/CreateNamedProjectStep.test'
+
+// Mocking done by selectManagedApp.
+jest.mock('src/libs/ajax')
+
+describe('transforming user info to the request object', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+
+  it('splits lists of emails and maps them to roles', () => {
+    const emailList = 'a@b.com, b@c.com'
+    const result = userInfoListToProjectAccessObjects(emailList, 'User')
+
+    expect(result).toHaveLength(2)
+    expect(result[0].role).toEqual('User')
+    expect(result[0].email).toEqual('a@b.com')
+    expect(result[1].role).toEqual('User')
+    expect(result[1].email).toEqual('b@c.com')
+
+    // Empty emails returns an empty array
+    expect(userInfoListToProjectAccessObjects('', 'User')).toHaveLength(0)
+  })
+})
+
+const testStepActive = stepNumber => {
+  screen.queryAllByRole('listitem').forEach((step, index) => {
+    if (index === stepNumber - 1) {
+      expect(step.getAttribute('aria-current')).toBe('step')
+    } else {
+      expect(step.getAttribute('aria-current')).toBe('false')
+    }
+  })
+}
+
+describe('AzureBillingProjectWizard', () => {
+  let renderResult
+  const onSuccess = jest.fn()
+  const captureEvent = jest.fn()
+
+  beforeEach(() => {
+    renderResult = render(h(AzureBillingProjectWizard, {
+      onSuccess
+    }))
+  })
+
+  it('should not fail any accessibility tests in initial state', async () => {
+    expect(await axe(renderResult.container)).toHaveNoViolations()
```

Comment:
Note that accessibility checks cannot be done on the individual step components because they fail due to li being outside a list.